core/src/Borrower.sol

```diff
@@ -140,21 +140,21 @@ contract Borrower is IUniswapV3MintCallback {
                 // If both are zero or neither is zero, there's nothing more to do.
                 // Callbacks/swaps won't help.
             } else if (liabilities0 > 0) {
-                uint256 converted0 = liabilities0 / strain;
-                uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96) + incentive1 / strain;
+                uint256 expected0 = liabilities0 / strain;
+                uint256 advance1 = Math.mulDiv(expected0, priceX96, Q96) + incentive1 / strain;
 
-                TOKEN1.safeTransfer(address(callee), maxLoss1);
-                callee.callback0(data, maxLoss1, converted0);
+                TOKEN1.safeTransfer(address(callee), advance1);
+                callee.swap1For0(data, advance1, expected0);
 
-                repayable0 += converted0;
+                repayable0 += expected0;
             } else {
-                uint256 converted1 = liabilities1 / strain;
-                uint256 maxLoss0 = Math.mulDiv(converted1 + incentive1 / strain, Q96, priceX96);
+                uint256 expected1 = liabilities1 / strain;
+                uint256 advance0 = Math.mulDiv(expected1 + incentive1 / strain, Q96, priceX96);
```

Comment:
Subject to potential change per https://github.com/aloelabs/aloe-ii/pull/30#discussion_r1073222041.